atc/metric/emitter/newrelic_test.go

```diff
@@ -0,0 +1,100 @@
+package emitter
```

Comment:
we usually don't have our tests sharing the same package as the package being tested, e.g.:

```console
$ ag -Q "package" **/*_test.go

scheduler/runner_test.go
1:package scheduler_test

scheduler/scheduler_suite_test.go
1:package scheduler_test
...
```

while this definitely restricts the surface that you can touch (only exported methods, ifaces, structs, vars, consts, etc) though, it'll make the tests be a more close approximation of "docs as code"